.github/workflows/ci.yaml

```diff
@@ -80,12 +82,12 @@ jobs:
         # https://github.com/instrumenta/kubernetes-json-schema is no
         # longer updated
         k8s:
-          # - v1.14.10 (deprecated with kind 1.16)
-          - v1.16.15
+          - v1.16.4
```

Comment:
note: https://github.com/instrumenta/kubernetes-json-schema doesn't seem to have 1.16.15; 1.16.4 is the latest. Presumably this is Good Enough, but if someone can find the right schema version (I'm surprised `kubeval` worked with this) online in the right format, I can update the secondary schema location 🤷